tests/test_train.py

```diff
@@ -1,3 +1,83 @@
 import pytest,fastai
+from utils.fakes import *
+from utils.text import *
 
-#I'm all empty, please fill me with tests about the functions in train.py
\ No newline at end of file
+## filename: test_train.py
+## tests functions in train.py
+
+## Class Learner, see documentation:  https://docs.fast.ai/train.html
+ 
+## Model fitting methods
+
+## run: pytest tests/test_train.py (add -s for screenoutput )
+
+
+@pytest.fixture(scope="module")
+def learn():
+    
+    learn = fake_learner(50,50)
+   
+    return learn
+
+def test_fit(learn):
+
+    ## Test confirms learning rate and momentum are stable, see difference to test_fit_one_cycle
+    
+    learning_rate = 3e-3
+    weight_decay = 1e-2
+    eps = 4
+    
+    with CaptureStdout() as cs:  learn.fit(epochs=eps, lr=learning_rate, wd=weight_decay)
+   
+    lrs = list(learn.recorder.lrs)
```

Comment:
I think you misunderstood my previous comment, we can test that we always used the same lr by checking `set(learn.recorder.lrs) == {learning_rate}`.  Same for the momentum!

Reply:
ah, okay. Yes - understood now and adjusted (less code indeed)